packages/flow/lib/AstroIterate.astro.js

```diff
@@ -1,37 +1,43 @@
-import { getNormalizedGenerator, isIterable } from './shared.js'
+import {
+	getNormalizedGenerator,
+	hasForEachMethod,
+	isIterable,
+} from './shared.js'
 
 export default Object.assign(
-	function Iterate(_result, attributes, slots) {
+	function Iterate(_result, { of: each }, slots) {
 		const promiseOfGenerator = Promise.resolve(
```

Comment:
I assumed attributes were props so I have changed it to the destructuring of the `of=` one. The of is the attribute that is being evaluated.